### ISSUE TITLE
Sometimes in the Print Preview, the last word or the last character of some centered title page values is put on its own line.

### ISSUE THREAD

**Issue by BFLTP** [lmparppei/Beat#77]
Examples:
```
Title: Beat
```
The `t` in `Beat` is put on its own line below `Bea` in the Print Preview.

```
Title: Beat
Author: Imparppei
```
`Beat` is formatted properly, but the `i` in `Imparppei` is put on its own line below `Imparppe`.

```
Title: Beat App
```
`App` is put on its own line below `Beat`.

```
Title: Beat App
Author: Lauri-Matti Parppei
```
`Parppei` is put on its own line. Everything else is formatted properly.

```
Title:
    Beat App
    Version 1.911
Credit: Written by
```
`1.911` is put on its own line below `Version`, but everything else is formatted properly.

The actual exported PDF does not have this issue.

**Comment by lmparppei** [comment]
This is probably an issue in `ScriptCSS.css`

**Comment by lmparppei** [comment]
To debug locally, you can edit the `ScriptCSS.css` file found inside app container (under `Resources`) and post your findings. I'll look into this when possible.

**Comment by lmparppei** [comment]
Fixed in `1.91.2`
